misc/python/materialize/mzcompose/__init__.py

```diff
@@ -623,15 +624,36 @@ def up(
             self.compose = old_compose
             self._write_compose()
 
-    def down(self, destroy_volumes: bool = True, remove_orphans: bool = True) -> None:
+    def sanity_restart_mz(self) -> None:
+        """Restart Materialized if it is part of the composition to find
+        problems with persisted objects, functions as a sanity check."""
+        if "materialized" in self.compose["services"]:
+            print("SANITY RESTART")
+            self.kill("materialized")
+            # TODO: Better way to detect when kill has finished
+            time.sleep(3)
+            self.up("materialized")
+            self.sql("SELECT 1")
```

Comment:
Can you please open a follow-up ticket to add more verification queries in the sanity restart part? For example:
- checks that all sources are in a state of "running"
- check that all replicas are in a state of "ready"